ui/npm-content/build.sh

```diff
@@ -58,3 +58,7 @@ rm -rf "$libsDirectory/tinymce"
 cp -r ../../node_modules/tinymce "$libsDirectory/tinymce"
 
 echo "TinyMCE build complete"
+
+# monaco build
+webpack --config webpack.config.js
```

Comment:
Will this line work in all cases? How? Webpack is typically not a global bash command. Should probably be ran through a package.json script via `yarn`.

Check if you have webpack installed globally.

Reply:
Moved webpack command to yarn script